doc/tutorial.rst

```diff
@@ -0,0 +1,131 @@
+Tutorial
+========
+
+This should get you started with using ``dnaio``.
+The only essential concepts to know about are
+the `dnaio.open` function and the `SequenceRecord` object.
+
+
+Reading
+-------
+
+The main interface for reading and writing sequence files is the `dnaio.open` function.
+For example, this program reads in a FASTQ file and computes the total number of nucleotides
+it contains::
+
+    import dnaio
+
+    with dnaio.open("reads.fastq.gz") as reader:
+        bp = 0
+        for record in reader:
+            bp += len(record)
+    print(f"The input file contains {bp/1E6:.1f} Mbp")
+
+As can be seen from the ``.gz`` file extension,
+the input file is gzip-compressed.
+`dnaio.open` detects and handles this automatically by opening the file with
```

Comment:
Using single backticks creates referencable text, but not code. Double backticks are needed in the latter case.

Do these references reference back to the API documentation?

Reply:
Yes, the single backticks were intentional. It worked for `dnaio.open` in the README, but now I see that it doesn’t work for `SequenceRecord` and all the other classes where I don’t have a `dnaio.` prefix.

Reply:
Ah, I need to write `~dnaio.SequenceRecord` to get what I want.